src/components/Greeting.jsx

```diff
@@ -0,0 +1,26 @@
+// app/javascript/components/Greeting.js
+import React, { useEffect } from 'react';
```

Comment:
- [ ] Please kindly remove this commented line of code in order to make your codebase clean. 👍

Reply:
Done, Thank you @OyePriscilla